### ISSUE TITLE
Crash in StoryRecyclerViewAdapter

### ISSUE THREAD

**Issue by jyn514** [hidroh/materialistic#1246]
The traceback had an IndexOutOfBounds exception in https://github.com/hidroh/materialistic/blob/master/app/src/main/java/io/github/hidroh/materialistic/widget/StoryRecyclerViewAdapter.java#L394. Maybe the number of items changes between when you call setUpdated and when DiffUtils calls the callback? This normally happens when I scroll up rapidly, but before I swipe down to refresh.

**Comment by hidroh** [comment]
Yes I'm aware of this and working for a fix. It's related to #1245
